Gems/LmbrCentral/Code/Tests/Builders/SeedBuilderTests.cpp

```diff
@@ -78,7 +78,7 @@ TEST_F(SeedBuilderTests, SeedBuilder_EmptySourceDependency_Valid)
 {
     DependencyBuilder::SeedBuilderWorker seedBuilderWorker;
     AssetBuilderSDK::CreateJobsRequest request;
-    constexpr char testSeedFolder[] = "@root@/../Gems/LmbrCentral/Code/Tests/Seed";
+    constexpr char testSeedFolder[] = "@projectproductassets@/../Gems/LmbrCentral/Code/Tests/Seed";
```

Comment:
Same here.

Reply:
Still needs an update here.

Reply:
I missed the second one in the file.
I will change it